lib/authn.js

```diff
@@ -91,7 +81,7 @@ async function _connect(options) {
     delete connectOptions.socketOptions;
   }
   const client = await MongoClient.connect(options.url, connectOptions);
-  const db = client.db(options.database);
+  const db = client.db();
   const ping = await db.admin().ping();
   logger.debug(
     'database connection succeeded: db=' + options.database +
```

Comment:
Does `options.database` still always match now? i.e., can this log output be wrong / confusing?

Reply:
good catch this should use the db object for the name in all cases.

Reply:
I corrected this here: https://github.com/digitalbazaar/bedrock-mongodb/pull/89/commits/881884e81b6cacfa403d8ffcc0ebe2c0f29dd254 but there might be other places so a more thorough review of the lib is needed.